README.md

```diff
@@ -107,12 +107,44 @@ Before deploying the application, you need to set the environment variables, as
 cp .env.example .env.production
 ```
 
+#### SSL certification
+
+To setup HTTPS, you will need valid SSL certificates. If you deploy the app for the first time, follow these instructions:
+- Comment or delete the whole server section about 443 in the `nginx/nginx.conf.template` file.
+```
+- server {
```

Comment:
what are the `-` at the beginning?
it does not look like a "diff", does it? :open_mouth: 

if it's meant to be a "diff", please write
```
```diff
- server {
- listen 443 default_server ssl http2;
- ...
```
to have the output colored in red :wink:

Reply:
Fix [1cfe0df](https://github.com/iScsc/iscsc.fr/pull/34/commits/1cfe0df6adf500938942d93b374f99ebf266bcc3)

Reply:
perfect thanks :yum: